slick/src/main/scala/slick/jdbc/JdbcActionComponent.scala

```diff
@@ -1,20 +1,15 @@
 package slick.jdbc
 
 import java.sql.{PreparedStatement, Statement}
-
-import scala.language.{existentials, higherKinds}
-import scala.collection.mutable.Builder
-import scala.util.control.NonFatal
-
 import slick.SlickException
 import slick.ast.ColumnOption.PrimaryKey
 import slick.dbio._
-import slick.ast._
+import slick.ast.{Comment, _}
```

Comment:
The old import includes `Comment` as well. There is no need to list it explicitly

Reply:
Changed to use _